tests/config_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 var (
 	configListCmd  = "config:list --app={{.AppName}}"
-	configSetCmd   = "config:set FOO=讲台 --app={{.AppName}}"
+	configSetCmd   = "config:set FOO=讲台` --app={{.AppName}}"
```

Comment:
Could we add a new test for this? I'd rather know that if this test fails, it's an issue with character set encoding and not the backtick escaping.